src/pages/MovieDetails.vue

```diff
@@ -15,44 +36,211 @@ export default defineComponent({
   },
   data() {
     return {
-      movieDetails: null,
-      isLoading: true,
-      errorMessage: null,
+      movie: null,
+      movieIsLoading: true,
+      movieError: null,
+      date: new Date(),
     };
   },
   computed: {
+    ...mapState(useSeancesStore, [
+      "seances",
+      "seancesIsLoading",
+      "seancesError",
+      "getSeancesErrorMessage",
+    ]),
+    getMovieErrorMessage() {
+      return (
+        this.movieError?.message ||
+        "We are sorry, but the movie cannot be displayed"
+      );
+    },
     getMovieTitle() {
-      return this.movieDetails.title;
+      return this.movie.title;
```

Comment:
You could define const `seancesStore = useSeancesStore()` then access your state and getters like this: `seancesStore.movie.title` this would save on creating computed properties for all data attributes that you need. 

```
<script>
import { useSeancesStore } from ‘@/store/seances’;
setup() {
    const seancesStore = useSeancesStore();
  return {
          seancesStore,
  }
}
</script>
<template>
  <h1>{{ seancesStore.movie.title }}</h1>
</template>
```
https://pinia.vuejs.org/core-concepts/state.html#accessing-the-state
https://pinia.vuejs.org/core-concepts/getters.html